homepage.html

```diff
@@ -0,0 +1,204 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Homepage</title>
+  <link rel="stylesheet" href="style.css" />
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" />
+  <link href="http://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet">
+  <link
+    href="https://www.dafontfree.net/embed/Y29jb2dvb3NlLXByby10aGluLXJlZ3VsYXImZGF0YS84NS9jLzE3NjcxMS9Db2NvZ29vc2UgUHJvIFRoaW4tdHJpYWwudHRm"
+    rel="stylesheet" type="text/css" />
+  <link href='http://fonts.googleapis.com/css?family=Lato:400,700' rel='stylesheet' type='text/css'>
+</head>
+
+<body>
+
+  <header class="header">
+
+    <img class="header-img" src="images/pexels-helena-lopes-933964.jpg" alt="cover image">
+
+    <div class="header-container">
+
+      <nav>
+        <div class="logo"><img src="images/logo.jpg" alt="logo">
+          <p>Nobilis</p>
+        </div>
+        <div class="openMenu"><img src="https://img.icons8.com/ios-filled/50/000000/menu--v1.png" alt="open menu" />
+        </div>
+        <ul class="mainMenu">
+          <li><a href="about.html">About Us</a></li>
+          <li><a href="#">Program</a></li>
+          <li><a href="#">Lecturers</a></li>
+          <li class="closeMenu"><img src="https://img.icons8.com/ios-glyphs/30/000000/delete-sign.png"
+              alt="close menu" /></li>
+        </ul>
+
+      </nav>
+
+      <div class="bootcamp-headings">
```

Comment:
- [ ] Please use semantic tags: sections use `section`, articles the nav and footer use `nav` and `footer`. Use `<section>` instead of `<div>`. Rember what the previous review said